pkg/membrane/membrane.go

```diff
@@ -191,7 +193,7 @@ func (s *Membrane) Start() error {
 	}
 	lis, err := net.Listen("tcp", s.serviceAddress)
 	if err != nil {
-		return fmt.Errorf("Could not listen on configured service address: %v", err)
+		return fmt.Errorf("could not listen on configured service address: %w", err)
```

Comment:
Could this be log.Default().Printf?

Reply:
it's returning an error. that should bubble up to a log at some point.